setup.py

```diff
@@ -12,9 +12,32 @@ def read_version(fname="whisper/version.py"):
 
 
 requirements = []
+whisper_rocm = os.getenv('WHISPER_ROCM',default='0')
 if sys.platform.startswith("linux") and platform.machine() == "x86_64":
-    requirements.append("triton==2.0.0")
+<<<<<<< HEAD
```

Comment:
this line is some leftover?

Reply:
Hi @vadimkantorov  I just rebased this fork to the latest main branch and also fixed the conflicts. Please review. Thanks!